pycloudlib/ec2/instance.py

```diff
@@ -294,6 +294,9 @@ def _attach_network_interface(self, interface_id):
                         'DeleteOnTermination': True
                     }
                 )
+                return nic.private_ip_address
+        raise Exception('Could not nic with AttachmentId: {}'.format(
+            response['AttachmentId']))
```

Comment:
Have we seen the failure case here? I'm wondering if 'AttachmentId' may not exist in failure cases and if we need to safe guard that assumption with a `.get`